src/components/recipe-timing-information.module.css

```diff
@@ -0,0 +1,34 @@
+@value colors: "../styles/colors.css";
+@value tuna from colors;
+
+.container {
+  min-width: 300px;
+  flex: 1;
+  border: 1px solid tuna;
+  border-radius: 4px;
+  text-align: center;
+  margin-top: 8px;
+  display: flex;
+  flex-direction: column;
+  padding: 8px;
+}
+
+.header {
+  margin-bottom: 4px;
+}
+
+.cells {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(125px, 1fr));
```

Comment:
Keep in mind that css grid support is worse than flexbox.

Reply:
I checked caniuse and nothing jumped out that would make this go horrible wrong 😂 It works on every device I'll be using lol I will keep this in mind when making other components 👍 Thanks!